youtube_dl/extractor/nest.py

```diff
@@ -0,0 +1,39 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+
+
+class NestIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?video.nest\.com/clip/(?P<id>)(.mp4)?'
```

Comment:
This will never match a useful ID!
```suggestion
    _VALID_URL = r'https?://(?:www\.)?video\.nest\.com/clip/(?P<id>\w+)'
```